### PR TITLE
Jet corrector parameters update for l1 residuals

### PR DESCRIPTION
**Pull request by aperloff**
The JERC group would like to rename two of the empty UncertaintyAux correction levels. This will allow us to include 1-2 new correction levels in the future. These are files/corrections that are being used and/or created in the JERC group now, and may become part of the recommended workflow in the future. Given that we are simply renaming two empty levels we do not foresee any forward/backward compatibility issues. @rappoccio @blinkseb

### CODE REVIEW

**Comment by monttj** [comment]
+1

**Comment by monttj** [comment]
please test

File: CondFormats/JetMETObjects/interface/JetCorrectorParameters.h [hunk 1]
```diff
@@ -149,8 +151,6 @@ class JetCorrectorParametersCollection {
 		 UncertaintyPileUpJetRate=26,
 		 UncertaintyAux1=34,
 		 UncertaintyAux2=35,
-		 UncertaintyAux3=36,
```
Comment (rappoccio) [inline CondFormats/JetMETObjects/interface/JetCorrectorParameters.h]: Alexx, you have to put these two (36 and 37) back. You just change "UncertaintyAux1" to "L1RC" and "UncertaintyAux2" to "L1Residual".

File: CondFormats/JetMETObjects/src/JetCorrectorParameters.cc [hunk 2]
```diff
@@ -364,8 +364,8 @@ const std::vector<std::string> labels_ = {
   "UncertaintyPileUpPtHF",
   "UncertaintyAux1",
   "UncertaintyAux2",
-  "UncertaintyAux3",
```
Comment (rappoccio) [inline CondFormats/JetMETObjects/src/JetCorrectorParameters.cc]: And these should be switched with "UncertaintyAux1" and "...2" also.

**Comment by aperloff** [comment]
I have followed Sal's directions (I believe) and have tested that I can again make and read an SQLite file with the modifications.

**Comment by rappoccio** [comment]
Also Alexx, you can check these by running "scram b runtests"

**Comment by ggovi** [comment]
@aperloff 
This PR includes a change in the CondFormats JetCorrectorParametersCollection, namely the type Level_t has been modified. Note that change of this type potentially breaks the access to existing data in the database. Can you please verify that no data instantiated with JetCorrectorParametersCollection::Level_t has been stored?

Thanks

Giacomo

**Comment by rappoccio** [comment]
To be clear : This just renames an enum. We developed this in the recent past to be "forward compatible"... note the name is just "UncertaintyAux1" which is a dummy placeholder. This will not break anything in the database.

**Comment by rappoccio** [comment]
@slava77 @monttj @ggovi Just a ping, is there an estimate for when we can get this in? It's very high priority for Run 2 commissioning and analyses.

**Comment by slava77** [comment]
@cmsbuild please test

@rappoccio 
Sal, this looks benign to me.
However, it's not my L2 area to be able to sign

**Comment by monttj** [comment]
+1

**Comment by ggovi** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**